documentation/src/docs/pravega-concepts.md

```diff
@@ -11,429 +11,312 @@ You may obtain a copy of the License at
 cluster for high availability.  
 
-The [Segment Store](segment-store-service.md) implements the Pravega data plane.
- Pravega Servers provide the API to read and write data in Streams.  Data
-storage is comprised of two tiers: Tier 1 Storage, which provides short term,
-low-latency, data storage,  guaranteeing the durability of data written to
-Streams and Tier 2 Storage providing longer term storage of Stream data.
- Pravega uses [Apache Bookkeeper](http://bookkeeper.apache.org/) to implement
-Tier 1 Storage and uses HDFS, Dell EMC's Isilon or Dell EMC's Elastic Cloud
-Storage (ECS) to implement Tier 2 Storage.  Tier 1 Storage typically runs within
-the Pravega cluster.  Tier 2 Storage is normally deployed outside the Pravega
-cluster.
-
-Tiering storage is important to deliver the combination of fast access to Stream
-data but also allow Streams to store a vast amount of data.  Tier 1 storage
-persists the most recently written Stream data.  As data in Tier 1 Storage ages,
+The [Segment Store](Segment-store-service.md) implements the Pravega data plane.
+Pravega servers provide the API to read and write data in Streams. Data storage is comprised of two tiers:
+- **Tier 1:** It provides short term, low-latency, data storage, guaranteeing the durability of data written to Streams. Pravega uses [Apache Bookkeeper](http://bookkeeper.apache.org/) to implement
+Tier 1 Storage.
+- **Tier 2:** It provides long term storage for Stream data. Pravega uses HDFS, Dell EMC's Isilon or Dell EMC's Elastic Cloud Storage (ECS) to implement Tier 2 Storage.
+
+Tier 1 Storage typically runs _within_ the Pravega cluster. Tier 2 Storage is normally deployed _outside_ the Pravega cluster. Tiering storage is important to deliver the combination of fast access to Stream
+data and also allows large data storage for Streams. Tier 1 storage
+persists on the most recently written Stream data. As data in Tier 1 Storage ages,
 it is moved into Tier 2 Storage.
 
 Pravega uses [Apache Zookeeper](https://zookeeper.apache.org/) as the
 coordination mechanism for the components in the Pravega cluster.  
 
-Pravega is built as a data storage primitive first and foremost.  Pravega is
-carefully designed to take advantage of software defined storage so that the
+Pravega is built as a data storage primitive first and foremost. Pravega is
+carefully designed to take advantage of software defined storage, so that, the
 amount of data stored in Pravega is limited only by the total storage capacity
-of your data center.  And like you would expect from a storage primitive, once
-data is written to Pravega it is durably stored.  Short of a disaster that
-permanently destroys a large portion of a data center, data stored in Pravega is never
+of the data center. Once the data is written to Pravega it is durably stored.  Short of a disaster, that
```

Comment:
I don't like the sentence that was written before (`Short of...`), but the new one's meaning is the opposite than the previous one. Please, rephrase as follows:
`Once an Event is written to Pravega, it is durably stored and replicated so it can survive permanent crashes of datacenter nodes`. (delete the next sentence starting by `Short of...`)

Reply:
Accepted suggestion